README.md

```diff
@@ -73,4 +73,9 @@ note: inventory will be needed to be edited for appropriate slave node IPs
  docker build --target production --tag todo-app:prod .
 
 Dev -   docker run --env-file .env todo-app:dev
-Prd -   docker run --env-file .env todo-app:prod
\ No newline at end of file
+Prd -   docker run --env-file .env todo-app:prod
+
+#####Run tests in docker via ci pipeline
+run the pipeline in github
+
+or docker run --env-file .env todo-app:test
```

Comment:
We should also add the instructions for building this image, and as above we should be able to use the `.env.test` file (or actually, no such file at all since you copy it into the image anyway!)